src/components/Header/header.js

```diff
@@ -1,11 +1,19 @@
+/**
+ *  Header Component
+ */
+
+// Dependencies
 import React, {Component} from 'react';
+import Search from '../Search/search';
 import ResponsiveUtils from '../Widgets/ResponsiveUtils/responsiveUtils';
+import ImageOptimize from '../../components/Widgets/ImageOptimize/imageOptimize'
+import Logo from '../Logo/logo';
+
+// Styles & Images
 import './header.scss';
-import Background from '../../../public/assets/images/walking-dog-bg.png';
+import { Button , Container, Row, Col } from 'reactstrap';
+
 
-var bgImage = {
-    backgroundImage: "url(" + Background + ")",
```

Comment:
Find another way to do this, This looks cheap.

Reply:
fixed